backend/src/services/model.ts

```diff
@@ -335,6 +342,21 @@ export async function updateModel(user: UserInterface, modelId: string, modelDif
   return model
 }
 
+async function checkCollaboratorAuthorisation(collaborators: CollaboratorEntry[]) {
```

Comment:
Can we add a check in here to ensure there's not duplicate collaborators in the list? It's not technically in scope for this ticket, so I'm happy for it to be spun out into a new ticket if necessary.